src/RouteRouter.js

```diff
@@ -141,8 +141,9 @@ class RouteRouter extends AbstractRouter {
                 })
 
             }
-            //res.set('Content-Type', 'application/json');
-            return JSON.stringify(possibleRoutes);
+            // res.set('Content-Type', 'application/json');
+            // return JSON.stringify(possibleRoutes);
+            return possibleRoutes
```

Comment:
If you don't want it stringified, please do it on all Routers. Thanks

Reply:
Also please include an explanation on why this change is happening

Reply:
The value for the data key was an array made into a string. You can return the array object, which makes it easier to iterate over the unwrapped data on the client-side.